src/Workspaces/Core/Portable/Formatting/Engine/AbstractFormatEngine.OperationApplier.cs

```diff
@@ -185,14 +185,31 @@ private bool ApplyForceLinesOperation(AdjustNewLinesOperation operation, int pai
                 return true;
             }
 
+            private int getIndentation(int pairIndex, TriviaData triviaInfo)
```

Comment:
@heejaechang We need this change not just for smart indentation but also during code generation which goes through the normal formatting path. In case of generating a newly generated method with a default method body, we would still like to have the beginning statement, method body, end statement in different lines.

I did think about changing the indentation delta(1 or 0 based on indent style) of the indentation operation but wont that be a change required in every place where we create the indentation(It might be easier but I am not sure)??